src/datastore/common.ts

```diff
@@ -256,7 +256,11 @@ export interface DataStoreUpdateData {
     nftEvents: DbNftEvent[];
     contractLogEvents: DbSmartContractEvent[];
     smartContracts: DbSmartContract[];
+    names: DbBNSName[];
+    namespaces: DbBNSNamespace[];
   }[];
+  names: DbBNSName[];
+  namespaces: DbBNSNamespace[];
```

Comment:
Only one set of `names` and `namespaces` should be needed for this interface right? It looks like the ones within the `txs` scope makes the most sense in case we need to correlate name data with their associated tx later on.

Reply:
Yes, I have removed these extra variables and these you have mentioned above in this [commit](https://github.com/blockstack/stacks-blockchain-api/pull/363/commits/f2712fcb1dd8ba558cdcb0a38def2fc3861f4a5c).